dialects/hir/src/transforms/spill.rs

```diff
@@ -37,6 +37,11 @@ impl Pass for TransformSpills {
         // We need to drop our mutable reference while computing the analysis results
         let function = unsafe { FunctionRef::from_raw(&*op) };
         drop(op);
+        if function.borrow().body().is_empty() {
```

Comment:
@bitwalker Is this the correct method and place for skipping function declarations?

Reply:
I'd probably use `is_declaration` when possible, which is a method on the `Symbol` trait, and since you have a `FunctionRef` there, all you need is to have the trait in scope. Internally though it just checks if the body region is empty, and more generally (i.e. with non-Symbol operations as well) that's what you'd have to do.

Reply:
Nice! Thanks! Done.